modules/apps/site-initializer/site-initializer-welcome/src/main/resources/site-initializer/layout-utility-page-entries/cookie-policy/page-definition.json

```diff
@@ -258,7 +258,7 @@
 															"widgetInstance": {
 																"widgetConfig": {
 																},
-																"widgetName": "com_liferay_object_web_internal_object_definitions_portlet_ObjectDefinitionsPortlet_[$OBJECT_DEFINITION_ID:NecessaryCookieEntry$]"
+																"widgetName": "com_liferay_object_web_internal_object_definitions_portlet_ObjectDefinitionsPortlet_[$OBJECT_DEFINITION_CLASS_NAME_SUFFIX:NecessaryCookieEntry$]"
 															}
```

Comment:
@vngoliveira same logic here, change from `OBJECT_DEFINITION_CLASS_NAME_SUFFIX` to `OBJECT_DEFINITION_PORTLET_ID`